### ISSUE TITLE
Please try **sp** from github shortly

### ISSUE THREAD

**Issue by rsbivand** [MikkoVihtakari/ggOceanMaps#6]
I'll push a version of **sp** to `rsbivand/sp` shortly with the `SRS_string="EPSG:XXXX"` case  working on PROJ 5.2.0 for archived and current repo **ggOceanMaps**. I've run revdep tests for 736 packages with PROJ 5.2.0 and GDAL 2.2.4, and found another affected package: https://github.com/jhollist/elevatr/issues/56. I need to re-run the revdeps (~ 5 hours on multiple cores) so it may be tomorrow before I push the patch. I'll comment here when I do.

**Comment by MikkoVihtakari** [comment]
Thank you, Roger. I believe that the new version will work with the [tests I have implemented for the most common operating systems](https://github.com/MikkoVihtakari/ggOceanMaps/actions/workflows/R-CMD-check.yaml).

Does `sp::CRS("EPSG:XXXX")` redirect the `projargs` to `SRS_string`? I have utilized the hack you made for the `CRS()` function that one can supply either "+init" or "EPSG" as the first and only argument (`projargs`) and the function detects automatically which PROJ one uses. Perhaps one should also test with `sp::CRS("EPSG:XXXX")` and `sp::CRS("+init=epsg:XXXX")`  in addition to `sp::CRS(SRS_string="EPSG:XXXX")`?
